docs/UserGuide.md

```diff
@@ -254,7 +254,13 @@ Examples:
 
 Adds an event to the event list.
 
-Format: `add -e d/DESCRIPTION at/DATE_TIME`
+Format: `add -e [d/DESCRIPTION] [at/DATE_TIME] [ap/CONTACT_INDEX_LIST]`
+
+1) `DESCRIPTION` = What you want the description of the event to be.
+1) `DATE_TIME` = Date and time of the event.
+1) `CONTACT_INDEX_LIST` = Indexes of contacts in the middle panel you want to add to the event.
+`*` is accepted as a way to add all the contacts from the panel. (See examples below)
+1) `[ap/CONTACT_INDEX_LIST]` is optional and can be omitted.
```

Comment:
this documentation style seems inconsistent with the rest of the UG.

Reply:
this has been fixed